crates/biome_service/src/file_handlers/css.rs

```diff
@@ -436,6 +436,7 @@ pub(crate) fn fix_all(params: FixAllParams) -> Result<FixFileResult, WorkspaceEr
             .with_only(&params.only)
             .with_skip(&params.skip)
             .with_path(params.biome_path.as_path())
+            .with_enabled_rules(&params.enabled_rules)
```

Comment:
This looks like it was an oversight in the `fix_all()` method, so I added these.